src/openproduct/producttypen/urls.py

```diff
@@ -44,19 +44,27 @@
 description = """
 Een Api voor Producttypen.
 
-Een Producttype is de definitie van een Product. hierin word alle relevante data opgeslagen zoals informatie teksten bedoeld voor klanten.
+Een Producttype is de definitie van een Product. hierin word alle relevante data opgeslagen zoals informatie over de aanvraag.
 
-Een Product (zie product api) is de instantie, hierin worden de specifieke gegevens van de instantie opgeslagen zoals bijvoorbeeld de gegevens van de eigenaar.
+Een product is een instantie van een producttype (zie producttypen api), In een product wordt onder andere de gegevens van de eigenaar, de nodige data voor het product en bijvoorbeeld de status vastgelegd.
 
-Een producttype valt onder een thema. Een thema kan onderdeel zijn van een ander thema via het attribuut 'hoofd_thema'.
+Een producttype valt onder één of meerdere thema. Een thema kan onderdeel zijn van een ander thema via het attribuut 'hoofd_thema'.
 
 Een producttype kan worden gelinkt één of meerdere locaties, organisaties en/of contacten.
 
 Daarnaast kunnen de volgende modellen per product worden aangemaakt:
```

Comment:
Not sure if modellen is the right term here, maybe something like `Daarnaast kunnen de volgende objecten gekoppeld worden met producttypen`? (or producten, not sure which model is related to which)